src/System.Private.Xml/tests/XmlSchema/XmlSchemaSet/TC_SchemaSet_AnyAttribute.cs

```diff
@@ -0,0 +1,206 @@
+﻿using Xunit;
+using Xunit.Abstractions;
+using System.IO;
+using System.Xml.Schema;
+
+namespace System.Xml.Tests
+{
+    public class TC_SchemaSet_AnyAttribute : TC_SchemaSetBase
+    {
+        private ITestOutputHelper _output;
+
+        public TC_SchemaSet_AnyAttribute(ITestOutputHelper output)
+        {
+            _output = output;
+        }
+
+        public bool bWarningCallback;
```

Comment:
I know that generally variables should be above constructor, but it looks that in that project are below constructors. Look at TC_SchemaSet_EnableUpaCheck or TC_SchemaSet_Misc. So my question is what has higher priority - project code style or general correct code style?